README.md

```diff
@@ -381,6 +381,7 @@ This repository contains a curated list of awesome open source libraries that wi
 * [DEAP](https://github.com/DEAP/deap) ![](https://img.shields.io/github/stars/DEAP/deap.svg?style=social) - A novel evolutionary computation framework for rapid prototyping and testing of ideas. It seeks to make algorithms explicit and data structures transparent. It works in perfect harmony with parallelisation mechanisms such as multiprocessing and SCOOP.
 * [DeepSpeed](https://github.com/microsoft/DeepSpeed) ![](https://img.shields.io/github/stars/microsoft/deepspeed.svg?style=social) - A deep learning optimization library (lightweight PyTorch wrapper) that makes distributed training easy, efficient, and effective.
 * [einops](https://github.com/arogozhnikov/einops) ![](https://img.shields.io/github/stars/arogozhnikov/einops.svg?style=social) - Flexible and powerful tensor operations for readable and reliable code.
+* [exo](https://github.com/exo-explore/exo) ![](https://img.shields.io/github/stars/exo-explore/exo.svg?style=social) - exo helps you run your AI cluster at home with everyday devices.
```

Comment:
From skimming through repo it sounds like serving & monitoring - e.g. sounds more similar to triton, than to something like ray / dash, let me know otherwise